examples/aws-presigned-url/serve.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+
+if(str_starts_with($_SERVER['REQUEST_URI'], '/main.js')) {
+    header("Content-Type: text/javascript; charset=UTF-8");
+    passthru('corepack yarn outputBundle');
+    return true;
+} else if (str_starts_with($_SERVER['REQUEST_URI'], '/uppy.min.css')) {
```

Comment:
Would probably be more accurate to get the file extension and then map that to a content type but it's only an example so 🤷‍♂️